docs/modules/integrate/pages/connectors.adoc

```diff
@@ -346,4 +358,11 @@ The Jet API supports more connectors than SQL.
 |hazelcast-jet
 |streaming
 |at-least-once
+
+|xref:integrate:vector-collection-connector.adoc[VectorSinks.vectorCollection]
+[.enterprise]*{enterprise-product-name}*
```

Comment:
`{enterprise-product-name}` is not resolving, is this intended?

Reply:
works for me in preview:

![image](https://github.com/hazelcast/hz-docs/assets/79092062/f4a4f46d-50cc-477e-9ec9-318a2c4ce1d5)